pom.xml

```diff
@@ -69,8 +69,8 @@
     <jaxb-api.version>2.3.3</jaxb-api.version>
     <jaxb.version>2.3.3</jaxb.version>
     <tomcat.version>9.0.38</tomcat.version>
-    <lucene.version>8.6.0</lucene.version>
-    <elasticsearch.version>7.9.0</elasticsearch.version> <!-- when changing this make sure JNA version is in sync -->
+    <lucene.version>8.6.2</lucene.version>
+    <elasticsearch.version>7.9.1</elasticsearch.version> <!-- when changing this make sure JNA version is in sync -->
```

Comment:
in the end, should be squashed with https://github.com/nuxeo/nuxeo/pull/4293/commits/f2afb3d1cc87dd9b40c7e2d50499c7af381dd55e#diff-600376dffeb79835ede4a0b285078036L72

Reply:
yes, I have updated now upgrade is on 7.9.2